ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/headervalidationrules/TimestampMoreRecentThanParent.java

```diff
@@ -44,11 +44,14 @@ private boolean validateTimestamp(final long timestamp, final long parentTimesta
 
   private boolean validateHeaderSufficientlyAheadOfParent(
       final long timestamp, final long parentTimestamp) {
-    if ((timestamp - minimumSecondsSinceParent) < parentTimestamp) {
+    final long actualTimeSinceParent = timestamp - parentTimestamp;
```

Comment:
not sure "actual" adds value - secondsSinceParent is probably closer.

Reply:
renamed